lib/src/chargebee.dart

```diff
@@ -10,11 +8,15 @@ import 'dart:convert';
 
 class Chargebee {
   static const platform = MethodChannel(Constants.methodChannelName);
+  static String status="";
+  static List<CBItem> listItems = [];
+  static List<Product> products = [];
+  static List<Subscripton> subscriptions = [];
+  static List<CBPlan> listPlans = [];
 
 /* Configure the app details with chargebee system */
-  static Future<void> configure(String site, String publishableApiKey,
+  static Future<String> configure(String site, String publishableApiKey,
```

Comment:
Can we define an enum for status?

Reply:
As discussed, status been removed on configure.